plugwise_usb/connection/queue.py

```
@@ -0,0 +1,156 @@
+        self._running = False
+        if self._submit_worker_task is not None and not self._submit_worker_task.done():
+            cancel_request = PlugwiseCancelRequest()
+            await self._submit_queue.put(cancel_request)
+            await self._submit_worker_task
+        self._submit_worker_task = None
+        self._stick = None
+        _LOGGER.debug("queue stopped")
+
+    async def submit(self, request: PlugwiseRequest) -> PlugwiseResponse | None:
+        """Add request to queue and return the received node-response when applicable.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)